solidity/.soliumrc.json

```diff
@@ -1,18 +1,16 @@
 {
-  "extends": "solium:recommended",
+  "extends": "keep",
```

Comment:
There are some rules that we use repetitievely across our repos, maybe we should review them and pull them into `solium-config-keep`. But this is not for the scope of this PR.

Reply:
https://github.com/keep-network/keep-core/issues/2303